pages/regions.js

```diff
@@ -0,0 +1,65 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { nanoid } from 'nanoid';
+
+import Layout from '../src/components/layout';
+import { Hero, Icon, P } from '../src/components/jambonz-ui';
+import { getData } from '../src/lib/data';
+
+function Regions({ data }) {
+  const router = useRouter();
+  const redirect = router.query.redirect;
+
+  return (
+    <>
+      {data.regions.map((region) => {
+        const href = (redirect === 'login') ? region.url : region.altUrl;
+
+        return (
+          <div key={nanoid()} region={region.name} className="wrap regions__wrap">
+            <div className="regions__icon">
+              <Link href={href}>
+                <a>
```

Comment:
So, personally I don't think we need the icon to be clickable since we have the text link with the external icon UI. This also creates a small a11y issue in Lighthouse: "Links do not have a discernible name". To fix this you can add a `title` attribute to each of these icon `<a>` elements. However, I would prefer we just nix them since the clickable UI is clearly marked with the text links 🤔